job-dsl-core/src/main/groovy/javaposse/jobdsl/dsl/DslFactory.groovy

```diff
@@ -143,6 +144,23 @@ interface DslFactory extends ViewFactory {
     WorkflowJob workflowJob(String name, @DslContext(WorkflowJob) Closure closure)
 
     /**
+     * Create or updates a workflow multi branch job.
+     *
+     * @since 1.41
+     * @see #workflowMultiBranchJob(java.lang.String, groovy.lang.Closure)
+     */
+    @RequiresPlugin(id = 'workflow-aggregator', minimumVersion = '1.11')
```

Comment:
The multibranch plugin is not part of the aggregator, so change the id to `workflow-multibranch`.